tests/unit/testsuite/Bolt/Boltpay/Controller/Traits/OrderControllerTraitTest.php

```diff
@@ -2,6 +2,7 @@
 
 require_once('TestHelper.php');
 require_once('CouponHelper.php');
+require_once('MockingTrait.php');
```

Comment:
@mccottry , please remove, if it possible.

Reply:
It is currently required.